.github/workflows/deploy.yml

```diff
@@ -7,7 +7,6 @@ on:
 permissions:
   contents: read
   pages: write
-  id-token: write
```

Comment:
You don't need to generate an OIDC token to deploy to GitHub pages

Reply:
So I merged this and.. fun fact
turns out we do
The deploy failed with `Error: Ensure GITHUB_TOKEN has permission "id-token: write".`

I have a bunch of repos where I write to the gh-pages branch of a repo without this permission, but having a quick scan over the docs for https://github.com/actions/deploy-pages doing it with actions/deploy-pages does it with an OIDC token :mortar_board:

Anyway.. I reverted this commit and pushed to master to fix.